helpers.php

```diff
@@ -416,7 +416,7 @@ function getAltapayChildOrderDetails($orderID)
         return false;
     }
 
-    $sql = 'SELECT * FROM `' . _DB_PREFIX_ . 'altapay_child_order` WHERE id_order =' . $orderID;
+    $sql = 'SELECT * FROM `' . _DB_PREFIX_ . 'altapay_child_order` WHERE unique_id =' . $orderID;
```

Comment:
Is $orderID safe?

Reply:
Actually, we are now using uniqueId instead of orderId to retrieve child order details. I have updated the variable name accordingly and sanitized it when passing it in the query.